apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingService.kt

```diff
@@ -156,28 +159,29 @@ class VilkaarsvurderingService(
         }
 
     private fun finnVilkaarForNyVilkaarsvurdering(
-        behandling: DetaljertBehandling,
         grunnlag: Grunnlag,
-        virkningstidspunkt: Virkningstidspunkt
-    ): List<Vilkaar> = when (behandling.sakType) {
+        virkningstidspunkt: Virkningstidspunkt,
+        behandlingType: BehandlingType,
+        sakType: SakType
```

Comment:
Brukes ikke?

Reply:
sakType brukes i when-en. Her har eg i grunn endra frå å ta inn heile DetaljertBehandling til å ta inn kun dei to relevante felta. Eigentleg mest for å kunne bruke ein DTO heller enn heile DetaljertBehandling, men uansett fint å tydeleggjera i signaturen kva denne metoden eigentleg treng, så lar det stå.